pkg/log/log.go

```diff
@@ -25,16 +26,35 @@ var (
 	)
 )
 
-// Init starts logging given the minimum log level
-func Init(logLevel string) error {
-	l := log.NewJSONLogger(log.NewSyncWriter(os.Stderr))
-	logLevelOption, err := parseLogLevel(logLevel)
+// Config represents a logger configuration used upon initialization.
+type Config struct {
+	Level  string
+	Format string
+}
+
+// ParseFlags parses the configuration flags for logging.
+func ParseFlags(cfg *Config) *Config {
+	flag.StringVar(&cfg.Level, "log-level", "debug", "The log level to use [ \"error\", \"warn\", \"info\", \"debug\" ].")
```

Comment:
How about: `Available log levels: "error", "warn", "info", "debug"`?

Reply:
IMO its clear enough how it was.